src/nvim/testdir/Makefile

```diff
@@ -61,6 +61,7 @@ NEW_TESTS ?= \
 	    test_timers.res \
 	    test_undo.res \
 	    test_usercommands.res \
+	    test_utf8.res \
```

Comment:
Upstream handles this by having a (I forget the real term for this) conversion rule in their Makefile:

```make
.vim.res:
	@echo "$(RUN_VIMTEST)" > vimcmd
	$(RUN_VIMTEST) $(NO_INITS) -S runtest.vim $*.vim
	@rm vimcmd
```

That allows all tests to be run independently while not automatically running them during general testing.